Formula/solc-select.rb

```diff
@@ -38,6 +39,11 @@ def install
     system bin/"solc-select", "install", "0.8.0"
     system bin/"solc-select", "use", "0.5.7"
 
+    assert_match(/0\.5\.7.*current/, shell_output("#{bin}/solc-select versions"))
```

Comment:
What defines this version? When will this need changed?

Reply:
The `use` command above this assert sets the version. It shouldn't need changing.

Reply:
Cool!